src/Generator/ValueGenerator.php

```diff
@@ -143,7 +149,7 @@ public function deleteConstant($constant)
     /**
      * Return constant list
      *
-     * @return ArrayObject
+     * @return SplArrayObject|StdlibArrayObject
```

Comment:
well, returning a SPL object in some cases is *already* a BC break (it is the only BC break btw).

If you keep using a mixed type for now, making the return type stricter in the future (to always use SPL) would be fine as it would not break any code using it, so the removal of StdLib return values could be done later in the future without BC break.